internal/e2e/cases_test.go

```diff
@@ -28,6 +28,38 @@ func runCases(c client, m *namespaceTestManager) func(*testing.T) {
 			assert.Len(t, resp.RelationTuples, 0)
 		})
 
+		t.Run("case=check subjectSet relations", func(t *testing.T) {
+			n := &namespace.Namespace{Name: t.Name()}
+			m.add(t, n)
+
+			obj := fmt.Sprintf("tree for client %T", c)
+			rel := "expand"
+
+			subjects := []string{"s1", "s2"}
+
+			for _, subjectID := range subjects {
+				c.createTuple(t, &ketoapi.RelationTuple{
+					Namespace: n.Name,
+					Object:    obj,
+					Relation:  rel,
+					SubjectID: &subjectID,
+				})
+			}
+
+			ss := &ketoapi.SubjectSet{
+				Namespace: n.Name,
+				Object:    obj,
+				Relation:  rel,
+			}
+			rt := &ketoapi.RelationTuple{
+				Object:     obj,
+				SubjectSet: ss,
+				Namespace:  n.Name,
+				Relation:   rel,
+			}
+
+			assert.True(t, c.check(t, rt))
```

Comment:
The reason why this test is failing is that the created tuples are not related to the checked tuple. It would be sufficient here to just create one tuple with a subject set and then check for the same tuple.